docs/section22/Rule22-11.md

```diff
@@ -0,0 +1,46 @@
+
+# CHW&CW - Rule 22-11  
+
+**Rule ID:** 22-11  
+**Rule Description:** For Baseline chilled-water system that does not use purchased chilled water, variable-flow secondary pump shall be modeled as 13W/gpm at design conditions.  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 22 CHW&CW Loop  
+**Appendix G Section Reference:** Section G3.1.3.10 Chilled-water supply temperature reset (System 7, 8, 11, 12 and 13)  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-7, 8, 11, 12, or 13.
+2. B-RMR is modeled with at least one chilled water system that does not use purchased chilled water.
+
+**Manual Check:** None  
+**Evaluation Context:** Building  
+**Data Lookup:** None  
+**Function Call:** None  
+
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-7, 8, 11, 12, or 13: `PLACEHOLDER`
+2. B-RMR is modeled with at least one chilled water system that does not use purchased chilled water: `if B_RMR.ASHRAE229.chillers:`
+
+## Rule Logic:  
+
+- For each chiller in B_RMR: `for chiller_b in B_RMR.ASHRAE229.chillers:`
+
+  - Get cooling loop that chiller servers: `chw_loop_b = chiller_b.cooling_loop`
+
+    - If cooling loop has not been saved in CHW loop array: `if chw_loop_b NOT in chw_loop_check_array:`
+
+      - Save cooling loop to CHW loop array that needs to comply with the requirement: `chw_loop_check_array.append(chw_loop_b)`
+
+- For each CHW loop that chiller serves: `for loop_b in chw_loop_check_array:`
+
+  - For each secondary loops served by CHW loop: `for secondary_loop_b in loop_b.child_loops:`
+
+    **Rule Assertion:**
+
+    - Case 1: If secondary pump is modeled as 13W/gpm at design conditions: `if secondary_loop_b.pump_power_per_flow_rate == 13: PASS`
+
```

Comment:
I think we should be looking at FluidLoop.pump_power_per_flow_rate and not parameters of individual pumps. This is similar to 21-9 RDS. The problem with using pump parameters is that there may be more than one pump serving the loop and we wouldn't be able to tell from the RMR whether these pumps are connected in paralles or series, which would affect how their power should be combined.

Reply:
pump power per flow rate will be in W/s-L and there should be a conversion from W/s-L to W/gpm

Reply:
As discussed previously, this type of conversion (IP to SI) will be handled in python code directly. Please confirm.